src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE users(
+    id int GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    first_name varchar(50) UNIQUE NOT NULL,
```

Comment:
Are you sure that we need a `unique` constraint on the `first_name` column?
That means two `Adam` or `Mark` won't be able to sign up in the application

Reply:
Yeah, you're right.